core/chains/evm/txmgr/broadcaster_test.go

```diff
@@ -1898,26 +1956,26 @@ func Test_IncrementNextNonce(t *testing.T) {
 	ethClient.On("PendingNonceAt", mock.Anything, addr1).Return(uint64(randNonce), nil).Once()
 	eb := NewTestEthBroadcaster(t, txStore, ethClient, ks, evmcfg, checkerFactory, false)
 
-	nonce, err := eb.GetNextSequence(addr1)
+	nonce, err := eb.GetNextSequence(testutils.Context(t), addr1)
```

Comment:
nit/ can reduce some noise by using one ctx per test:
```suggestion
	ctx := testutils.Context(t)
	nonce, err := eb.GetNextSequence(ctx, addr1)
```

Reply:
I've reduced the noise across a few different tests in the latest commit.